models/productDao.js

```diff
@@ -0,0 +1,35 @@
+const database = require("./dataSource");
+
+const getProducts = async (offset, limit, gender) => {
+  try {
+    const numberOffset = +offset;
+    const numberLimit = +limit;
+    return await database.query(
+      `
+    SELECT
+        p.id,
+        p.name,
+        p.price,
+        p.thumbnail_image_url AS thumbnail_url,
+        mc.name AS gender,
+        sc.name AS category
+    FROM products p
+    INNER JOIN sub_categories sc 
+    ON p.sub_category_id = sc.id
+    INNER JOIN main_categories mc 
+    ON sc.main_category_id = mc.id
+    WHERE mc.name = ? 
+    AND p.id LIMIT ?,? 
+    `,
+      [gender, numberOffset, numberLimit]
+    );
```

Comment:
이렇게 수정할 수 있을 것 같네요.
해당 sql문은 return 문에 포함되기 때문에 return문보다 들여쓰기를 하여 해당 코드의 일부분임을 표현할 수 있습니다.

또한 alias 지정 방식도 통일해주면 좋을 것 같아요!

```suggestion
    return await database.query(`
      SELECT
        p.id,
        p.name,
        p.price,
        p.thumbnail_image_url as thumbnail_url,
        mc.name as gender,
        sc.name as category
      FROM products as p
      INNER JOIN sub_categories as sc 
      ON p.sub_category_id = sc.id
      INNER JOIN main_categories as mc 
      ON sc.main_category_id = mc.id
      WHERE mc.name = ? 
      AND p.id LIMIT ?,?`, [gender, numberOffset, numberLimit]
    );
```

Reply:
수정 완료했습니다!